crates/valence_instance/src/lib.rs

```diff
@@ -447,6 +470,12 @@ pub struct PartitionCell {
     /// A cache of packets to send to all clients that are in view of this cell
     /// at the end of the tick.
     pub packet_buf: Vec<u8>,
+    /// A cache for packets that are layer specifics.
+    #[doc(hidden)]
+    pub layers_packet_buf: Vec<Vec<u8>>,
+    /// Mapping from layer to index in `layers_packet_buf`.
+    #[doc(hidden)]
+    pub layers_packet_buf_indices: FxHashMap<LayerType, usize>,
```

Comment:
Why do we need a Map between LayerType and a Vec of packet buffer instead of something like `FxHashMap<LayerType, Vec<u8>>`?